packages/grpc-js/src/resolver-dns.ts

```diff
@@ -88,6 +96,8 @@ class DnsResolver implements Resolver {
   private defaultResolutionError: StatusObject;
   private backoff: BackoffTimeout;
   private continueResolving = false;
+  private nextResolutionTimer: NodeJS.Timer;
+  private isNextResolutionTimerRunning = false;
```

Comment:
This almost feels like I'd want a special timer wrapper that'd add this property. Almost.

Reply:
`BackoffTimeout` actually does this, but we're not exponentially backing off here so it felt wrong to use it.